actionview/lib/action_view/helpers/asset_tag_helper.rb

```diff
@@ -364,6 +375,16 @@ def multiple_sources_tag_builder(type, sources)
           end
         end
 
+        def resolve_image_source(source, skip_pipeline)
+          if source.is_a?(Symbol) || source.is_a?(String)
```

Comment:
`NoMethodError` is a sign that whatever what passed into `polymorhic_url` is neither a model nor has a `resolve` for it.